generate-certs/main.tf

```diff
@@ -0,0 +1,152 @@
+# openvpn-generate-certs
+
+provider "aws" {
+  region = "${var.region}"
+}
+
+/* ---------------------------- */
+/* IAM Role & Instance Profile  */
+/* ---------------------------- */
+
+# TODO: figure out how to de-dup
```

Comment:
noticing a number of these items for the iam profiles. There has to be a cleaner way to de-dup this.